web/src/app/app.component.html

```diff
@@ -83,7 +83,7 @@
       </div>
       <div fxHide.gt-xs fxFlex="100" fxLayout="row" fxLayoutAlign="space-between center" class="app-toolbar">
         <div>
-          <mat-icon svgIcon="logo_icon" class="app-toolbar__logo"></mat-icon>
+          <img src="assets/icons/Logo.svg" class="app-toolbar__logo"/>
```

Comment:
I think there should be a space before the close tag

```html
<img src="assets/icons/Logo.svg" class="app-toolbar__logo" />
```

Reply:
done